hc/front/forms.py

```diff
@@ -18,26 +18,26 @@ def clean_tags(self):
 
 
 class TimeoutForm(forms.Form):
-    timeout = forms.IntegerField(min_value=60, max_value=2592000)
-    grace = forms.IntegerField(min_value=60, max_value=2592000)
-
+    timeout = forms.IntegerField(min_value=60, max_value=31104000)
```

Comment:
What do you think of declaring these values in a variable outside of the `timeout` variable instantiation.

So you have 

```python
min_val = 60;
max_val = 31104000;

timeout = forms.IntegerField(min_value=min_val, max_value=max_val)
```

This reduces the repetition of those values within the code and makes it easier to maintain.